docs/sources/panels/queries.md

```diff
@@ -82,25 +82,25 @@ Panel data source query options:
 
 - **Max data points -** If the data source supports it, sets the maximum numbers of data points for each series returned. If the query returns more data points than the max data points setting, then the data source consolidates them (reduces the number of points returned by aggregating them together by average or max or other function).
   
-  There are two main reasons for limiting the number of points, performance and smoothing the line. The default value is the width (or number of pixels) of the graph as there is no point in having more data points than the graph panel can display.
+  There are two main reasons for limiting the number of points, performance and smoothing the line. The default value is the width (or number of pixels) of the graph as there is no point in having more data points than, what the graph panel can display.
```

Comment:
Please undo this change.

Reply:
i think the change makes the document better in terms of readability, if we do not make the change, the document might look like eating on words

Reply:
Sorry, but I don't agree that this change improves the readability.